content/influxdb/v2.0/reference/flux/language/types.md

```diff
@@ -82,6 +89,10 @@ Flux supports [RFC3339 timestamps](/influxdb/v2.0/reference/glossary/#rfc3339-ti
 - `YYYY-MM-DDT00:00:00Z`
 - `YYYY-MM-DDT00:00:00.000Z`
 
+| Minimum timestamp      | Maximum timestamp     |
+| -----------------      | -----------------     |
+| `-9223372036854775806` | `9223372036854775806` |
+
```

Comment:
These limits don't actually apply to the Flux `time` type. These only apply to the nanosecond epoch timestamps (numeric) in line protocol. You can remove this table.

```suggestion
```